jest.js

```diff
@@ -0,0 +1,15 @@
+const pluginJest = require('eslint-plugin-jest');
+
+// Configuration for Jest test files only.
+module.exports = [
+    {
+        files: ['**/*.spec.{js,ts}', '**/*.test.{js,ts}'],
+        ...pluginJest.configs['flat/recommended'],
+        languageOptions: {
+            globals: pluginJest.environments.globals.globals,
```

Comment:
duh, why did they make it `globals.globals`? 👀

Reply:
sorry, I copy pasted it from another PR, I don't think it's needed, will validate.